windows-driver-docs-pr/image/--hello-world---installation-file.md

```diff
@@ -71,11 +71,6 @@ hellowld.dll=1
 [SourceDisksNames.x86]
```

Comment:
This INF only has a single Models section of [Models.NTamd64] so this INF only applies on amd64 machines.  Since this INF only applies to a single architecture, you can drop the .x86 on SourceDisksFiles and SourceDisksNames. If you really want to keep the architecture there, then it should be .amd64